src/common/support.js

```diff
@@ -168,7 +168,7 @@ function getSupportInfo(version, opts) {
     version = currentVersion;
   }
 
-  const plugins = loadPlugins();
+  const plugins = loadPlugins(opts);
```

Comment:
This `opts` is not the same `options` (i.e. the context) we're passing around in other places. This is type `{showUnreleased: boolean}`.

This must have gotten mixed up in the merge conflicts of the PR

Reply:
Kind of - in https://github.com/prettier/prettier/pull/3536/files#diff-7ac67a485f01c26ab36e763ce88748a5 I switched it to use options but it wasn't required. Added `plugins` to the options that can be passed here.